example/android/app/build.gradle

```
@@ -11,9 +11,9 @@ react {
     //   The root of your project, i.e. where "package.json" lives. Default is '../..'
     // root = file("../../")
     //   The folder where the react-native NPM package is. Default is ../../node_modules/react-native
-    // reactNativeDir = file("../../node_modules/react-native")
+    reactNativeDir = file("../../../node_modules/react-native")
+    codegenDir = file("../../../node_modules/@react-native/codegen")
     //   The folder where the react-native Codegen package is. Default is ../../node_modules/@react-native/codegen
```

Let's restore the original order of these two lines
```suggestion
    //   The folder where the react-native Codegen package is. Default is ../../node_modules/@react-native/codegen
    codegenDir = file("../../../node_modules/@react-native/codegen")
```